src/tr2/game/gameflow/gameflow_new.c

```diff
@@ -22,7 +23,10 @@ static void M_LoadObjectString(const char *const key, const char *const value)
 {
     const GAME_OBJECT_ID object_id =
         ENUM_MAP_GET(GAME_OBJECT_ID, key, NO_OBJECT);
-    if (object_id != NO_OBJECT) {
+    if (object_id == NO_OBJECT) {
+        LOG_ERROR("Invalid object id: %s", key);
+    } else {
+        LOG_ERROR("Renaming object id %s to %s", key, value);
```

Comment:
Just checking if this was only for debugging as it produces a lot of additional logging. If not, should it be `LOG_INFO`?

Reply:
Yeah let's not check this in.

Reply:
Fixed